applications/server/modules/access.py

```diff
@@ -0,0 +1,83 @@
+from gluon import current
+from gluon import auth
+
+
+# Permission types.
+# v = view
+# a = admin (valid only for one whole device)
+# e = edit settings of procedure
+
+def register_device(name=None, description=''):
+    db = current.db
```

Comment:
Can you add some comment on what this function does?  
Also, this is nice, but it's not necessarily an access function.  It is a mix of a new device registration, 
and of the access function claiming that device to a user.  Perhaps the two should be separate. 
Perhaps this function should be done by the UI team.  Leave it here with a TODO: perhaps useful to the UI team.